src/pages/leeszaal/[readingRoomSlug]/index.tsx

```diff
@@ -88,10 +96,31 @@ const ReadingRoomPage: NextPage = () => {
 	 */
 
 	const { data: mediaResultInfo } = useGetMediaObjects(
-		{
-			query: (query.search || []).join(' '),
-			format: (query.format as ReadingRoomMediaType) || READING_ROOM_QUERY_PARAM_INIT.format,
-		},
+		[
+			{
+				field: MediaSearchFilterField.QUERY,
+				operator: MediaSearchOperator.CONTAINS,
+				value: query.search !== null ? query.search?.toString() : '',
+			},
```

Comment:
Only add the filter if the value is defined
```suggestion
			...(query.search?.toString()?.trim() ? [{
				field: MediaSearchFilterField.QUERY,
				operator: MediaSearchOperator.CONTAINS,
				value: query.search.toString(),
			}] : []),
```

Reply:
This is handled in https://github.com/viaacode/hetarchief-client/blob/feature/ARC-663/src/modules/media/services/media/media.service.ts#L26-L47